docs/basics/userguide/export_training_data.md

```diff
@@ -163,3 +163,265 @@ In the default writer you have the following information :
 - dw_time_elapsed : Time elapsed since writer initialization
 - global_step : Step at which the value was added.
 - n_simu : Added by {mod}`rlberry.manager.read_writer_data`, n_simu is an integer identifying the agent (if you use fit>1, you will have information on more than 1 agent in your writer.)
+
+
+## How to import data from tensorboard?
+
+Maybe you want to use other tools to train your agents, but you want to use rlberry tools for visualisation and/or statistical tests.
+If your training is compatible with tensorboard, you can load the data in a pandas dataframes to use them in rlberry. To do that, you can use the tool [tensorboard_folder_to_dataframe](rlberry.manager.tensorboard_folder_to_dataframe).
+
+Be careful about this 2 things:
+- The folder containing tensorboard results must respect the following tree structure :
+    `<tensorboard_log_folder/algo_name/n_simu/events.out.tfevents.xxxxx>`
+- You must have only one file (event.out.tfenvent.xxx) by leaf folder(n_simu), only the first one would be import !
+
+For this tuto you can do these training with stablebaseline, and log with tensorboard :
```

Comment:
For instance, suppose you do the following training with ...

Reply:
done !